ansible/roles/mirrors/tasks/redhat.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Ensure repo tools present
+  ansible.builtin.package:
+    name: dnf-plugins-core
+    state: present
+  when: ansible_pkg_mgr == 'dnf'
+
+- name: Add Princeton Rocky repo (preferred by low cost)
```

Comment:
I don't understand how cost comes into it . . . would we otherwise pay a fee for using a package repo?

Reply:
network cost (latency). 

“Low cost” is a gentle preference knob in DNF/YUM repo configs.

In a .repo file, the cost= setting assigns a numeric “penalty” to a repo (default ~1000).

When the same package/version is available from multiple enabled repos, DNF/YUM will prefer the repo with the lowest cost.

If that repo is slow or unavailable, it will automatically fall back to others (it doesn’t disable them).